src/overseer/store/jdbc.clj

```diff
@@ -0,0 +1,194 @@
+(ns ^:no-doc overseer.store.jdbc
+  "Implementation of overseer.core/Store for JDBC"
+  (:require [clojure.java.jdbc :as j]
+            [clojure.set :as set]
+            [clojure.string :as string]
+            [framed.std.core :as std]
+            (loom graph)
+            (overseer
+              [core :as core]
+              [util :as util])))
+
+(defn- in-placeholders
+  "Generate the appropriate number of `?` placeholders for an IN clause
+  for values `vs`"
+  [vs]
+  (string/join "," (repeat (count vs) "?")))
+
+(defn- compare-and-swap
```

Comment:
This is a pretty central function to this PR and correctness is super important. I'm going to investigate optimistic concurrency  / lock versioning schemes here which may be more robust.